라이브러리 설치 및 제거 && Device 모델 추가

cafitac opened the pull request:
member/setting.py
 - DEL: INSTALLED_APPS에서 fcm_django 삭제

requirements.txt
 - DEL: fcm-django 삭제

push/models.py
 - ADD: Device 모델 추가

push/migrations
 - ADD: Device model에 대한 migrations 파일 추가

hanseongd commented on scheduler/push/models.py:
```diff
@@ -1,3 +1,22 @@
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    registration_id = models.CharField(max_length=400)
+    active = models.BooleanField(default=True)
+    device_type = models.CharField(max_length=7, choices=DEVICE_CHOICES, null=True, blank=True)
```
Device 모델에서 device tpye이 null을 허용해준 이유가 뭐야??
blank는 False 여야 하지 않아?

cafitac replied:
device는 필수가 아니라 선택 사항으로 넣어도 상관이 없다구 생각되서 null이랑 blank 모두 true로 설정했어요 프론트 개발하시는 형들이랑 말도 해봐야할 것 같긴한데....
어떻게 할까요??

hanseongd replied:
device가 필수 사항은 아니니까 user의 device모델 object가 꼭 만들어질 필요가 없다는 뜻아니야?
blank는 빈문자열도 들어갈 수 있도록 설정해주는건데 만들어질 경우 빈문자열이 들어가면 안되니까 null만 True옵션을 줘야할 것 같아

cafitac replied:
찾아봤는데 문자열 관련 필드는 null=true를 쓰는 것보다 blank만 true로 설정해놓는게 좋다고 하는데 null에 대한 옵션은 지우고 blank만 남길까요??

hanseongd replied:
null설정안하면 무조건 값을 넣어줘야하는건데 이미 object가 존재한다면 빈값으로라도 넣어줘야하고
그전에 내가 말했던 사항이 맞다면 둘다 지워야돼

Commit:
라이브러리 설치 및 제거 && Device 모델 추가

member/setting.py
 - DEL: INSTALLED_APPS에서 fcm_django 삭제

requirements.txt
 - DEL: fcm-django 삭제

push/models.py
 - ADD: Device 모델 추가

push/migrations
 - ADD: Device model에 대한 migrations 파일 추가
